django/conf/urls/__init__.py

```diff
@@ -1,9 +1,17 @@
 from django.urls import include
 from django.views import defaults
 
-__all__ = ["handler400", "handler403", "handler404", "handler500", "include"]
+__all__ = [
+    "handler400",
+    "handler403",
+    "handler404",
+    "handler413",
+    "handler500",
+    "include",
+]
 
 handler400 = defaults.bad_request
 handler403 = defaults.permission_denied
 handler404 = defaults.page_not_found
+handler413 = defaults.content_too_large
```

Comment:
IMO, this is not an acceptable solution, we cannot add handlers for all kind of 400-500 error codes.

Reply:
An alternative approaches:

* change [resolve_error_handler](https://github.com/django/django/blob/73df8b54a2fab53bec4c7573cda5ad8c869c2fd8/django/urls/resolvers.py#L731) to return a generic error handler
* overwrite the status code on the response [here](https://github.com/django/django/blob/73df8b54a2fab53bec4c7573cda5ad8c869c2fd8/django/core/handlers/exception.py#L137) without changing anything else

On second thought I prefer the latter (as it has the smallest impact).

Reply:
I think a generic error handler is probably the way, though it needs some thought and design. Which errors should be passed to the handler? How will that change over time? And how can we provide a clean backwards-compatibility or deprecation path?  There’s already an inconsistency that `handler500` doesn’t receive the exception object, which we’d hopefully correct as part of this...

Reply:
My only argument against this is, that another solution with a one-line change for the tracker issue exists and a generic handler is probably more a task for a core contributor.

In other words: we could fix the issue without fixing the design problem.

If you agree that the one-line patch is better than nothing I can provide that, otherwise I would unassign the tracker issue so that a core member can pick it up to deal with design considerations.

Reply:
There’s absolutely no limitation on “core” contributors proposing design decisions! Anyone can do that, we’d just want consensus on a forum thread for something major. You for sure have the power to do this, if you’re willing to dive in and spend the time. It would be much appreciated.